### PR TITLE
config: enable prow for tikv

### PR DESCRIPTION
**Pull request by hi-rustin**
/hold

Part of https://github.com/ti-community-infra/configs/issues/63

### CODE REVIEW

**Comment by hi-rustin** [comment]
@andylokandy @zhouqiang-cl PTAL~

**Review by zhouqiang-cl — APPROVED** [review]

File: prow/config/config.yaml [hunk 1]
```diff
@@ -27,6 +27,48 @@ branch-protection:
...
+                strict: true
+            release-4.0:
+              enforce_admins: false
+              protect: true
+              required_status_checks:
+                contexts:
+                  - "DCO"
+                  - "idc-jenkins-ci/test"
+                strict: true
+            release-5.0-rc:
```
[the remark this answers is not in the document]
Comment (hi-rustin) [reply prow/config/config.yaml:64]: Maybe we can improve it in the future.

**Comment by hi-rustin** [comment]
@andylokandy PTAL～ 
Thanks!

**Comment by hi-rustin** [comment]
@Mini256  PTAL~ Thanks!

**Comment by hi-rustin** [comment]
/hold 

Waiting @Mini256  approve.

**Review by Mini256 — APPROVED** [review]

**Pull request merged**